nservicebus/azure-service-bus/keep-connectionstring-safe.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Keep connection string safe
+summary: Configuring Azure ServiceBus transport to secure connection strings. 
+component: ASB
+tags:
+- Cloud
+- Azure
+- Transports 
+- Security
+reviewed: 2016-04-26
+---
+
+include: keep-connection-string-safe
+
+To enable this behavior, apply the following configuration:
```

Comment:
Putting my non-maintainer :tophat: on. 

>To enable this behavior, apply the following configuration:

What is this behaviour? What does it do? Why would I wan to use it?
This is kind of information I'd like to find in the first paragraph and after that see _how_ to do it :)